### ISSUE TITLE
module.js:549 error when running hpts

### ISSUE THREAD

**Issue by noobslayer404** [oyyd/http-proxy-to-socks#4]
i am using ubuntu 18, after installing node.js and npm through:
sudo apt-get install node.js
sudo apt-get install npm

and i run the command in terminal:
hpts 
I get this error:
![screenshot from 2018-07-16 10-40-25](https://user-images.githubusercontent.com/26854386/42747333-af0d69c8-88e4-11e8-97cc-550dd56b4efa.png)

**Comment by oyyd** [comment]
What's the version of your node? I guess `sudo apt-get install npm` is unnecessary which may cause your problem.

**Comment by polyamide** [comment]
Just install the lastest version (`1.1.1`), and I have the same problem.

My node version is `v11.15.0` and npm is `6.7.0`. They're installed by nvm.
After looking at `node_modules/http-proxy-to-socks`, it seems the `lib` directory is missing.

In my case, install an older version (`1.1.0`) works.

**Comment by oyyd** [comment]
@polyamide You are right. `http-proxy-to-socks@1.1.1` is broken.. This is fixed in `http-proxy-to-socks@1.1.2`.
